pennylane/__init__.py

```diff
@@ -207,7 +203,7 @@ def jacobian(func, argnum):
     # pylint: disable=no-value-for-parameter
     if isinstance(argnum, int):
         return _jacobian(func, argnum)
-    return lambda *args, **kwargs: numpy.stack(
+    return lambda *args, **kwargs: _np.stack(
```

Comment:
Good question, I can't remember why I made this change (this was back when I was still working on the ADR).

Reply:
Oh, I remember the motivation behind this change. Essentially, `pennylane.__init__` shouldn't be importing the wrapped version of NumPy, as it places a dependence on Autograd that is not needed. In addition, the output of `qml.jacobian` is not differentiable anyway, since we haven't implemented second derivatives.